plugins/filters/askAI/askAI.cm

```diff
@@ -19,7 +19,7 @@ automations:
     on:
       - label_added
     if:
-      - {{ pr.labels | match(term="/ask-ai qa") | some }}
+      - {{ pr.labels | includes(term="/ask-ai qa") }}
```

Comment:
This change is incorrect. `includes` expect String input and not List of Strings, while `match` does

Reply:
I don't find any documentation about `match` or `includes`.
But I tested it multiple times in different ways, and it worked.

Reply:
The fact it’s working is because JS is very relaxed about types but the docs should show the right usage to avoid getting to corner cases. The reference is here http://docs.gitstream.cm/filter-functions/#match and http://docs.gitstream.cm/filter-functions/#includes

Reply:
got you, thanks